Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,50 +1,41 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton'
 
 // The row-child components
 
-function RowChild (props) {
-  const { className, text, bkg } = props;
-
-  return (
-    <div className={className} style={{backgroundColor: `${bkg}`}} >
-    {text}
-    </div>
-  );
-}
-
-
 const App = () => {
+
   return (
     <div className="mainContainer">
       <div className="row">
-        <RowChild className="row-child full" text="0" bkg="black" />
+          <NumberButton className="row-child full" text="0" bkg="black" />
       </div>
       <div className="row">
-        <RowChild className="row-child TF small" text="clear" />
-        <RowChild className="row-child forth white" text="/" bkg="red"/>
+        <NumberButton className="row-child TF small" text="clear" />
+        <NumberButton className="row-child forth white" text="/" bkg="red"/>
       </div>      
       <div className="row">
-        <RowChild className="row-child forth" text="7" />
-        <RowChild className="row-child forth" text="8" />
-        <RowChild className="row-child forth" text="9" />
-        <RowChild className="row-child forth white" text="X" bkg="red" />
+        <NumberButton className="row-child forth" text="7" />
+        <NumberButton className="row-child forth" text="8" />
+        <NumberButton className="row-child forth" text="9" />
+        <NumberButton className="row-child forth white" text="X" bkg="red" />
       </div>      
       <div className="row">
-        <RowChild className="row-child forth" text="4" />
-        <RowChild className="row-child forth" text="5" />
-        <RowChild className="row-child forth" text="6" />
-        <RowChild className="row-child forth white" text="-" bkg="red" />
+        <NumberButton className="row-child forth" text="4" />
+        <NumberButton className="row-child forth" text="5" />
+        <NumberButton className="row-child forth" text="6" />
+        <NumberButton className="row-child forth white" text="-" bkg="red" />
       </div>      
       <div className="row">
-        <RowChild className="row-child forth" text="1" />
-        <RowChild className="row-child forth" text="2" />
-        <RowChild className="row-child forth" text="3" />
-        <RowChild className="row-child forth white" text="+" bkg="red" />
+        <NumberButton className="row-child forth" text="1" />
+        <NumberButton className="row-child forth" text="2" />
+        <NumberButton className="row-child forth" text="3" />
+        <NumberButton className="row-child forth white" text="+" bkg="red" />
       </div>      
       <div className="row">
-        <RowChild className="row-child TF" text="0" />
-        <RowChild className="row-child forth white" text="=" bkg="red"/>
+        <NumberButton className="row-child TF" text="0" />
+        <NumberButton className="row-child forth white" text="=" bkg="red"/>
       </div>      
     </div>
   );
```

Comment:
**Day 2 Review**
- Great job tackling Project calculator Haja. You manage to implement the design, its almost looking like the provided file.
- Next steps i would take is to try and make it possible to implement event listeners for buttons when they are pressed and add some basic functionality. Switch case in JS might be handy for this.
- For future improvements try and use  .map() generate new components and pass props to them. This is how You create reusable components.
Overall great job 👍